acceptance/setup/common/005_SyncTime.rb

```diff
@@ -4,4 +4,8 @@
 # Note: QENG-3641 would allow beaker to take care of this for us
 test_name 'Ensure hosts have synchronized clocks'
 
-timesync(hosts, {:logger => logger})
+# QENG-3786 - OSX is already running ntpd and manually running ntpdate will return 1
+applicable_hosts = hosts.select{|host| host['platform'] !~ /osx/}
+if applicable_hosts then
```

Comment:
The intention is to avoid passing beaker an empty array; as it generally doesn't validate its arguments before attempting to use them.
You are correct that the if statement is useless as it always resolves as true - I'll update to have it test on Array.empty? instead